services/director-v2/tests/unit/with_dbs/test_route_computations_tasks.py

```diff
@@ -0,0 +1,182 @@
+# pylint: disable=redefined-outer-name
+# pylint: disable=unused-argument
+# pylint: disable=unused-variable
+
+
+from typing import Any, Callable, NamedTuple
+from unittest import mock
+
+import httpx
+import pytest
+from faker import Faker
+from fastapi import FastAPI, status
+from models_library.projects import ProjectAtDB, ProjectID
+from models_library.projects_nodes_io import NodeID
+from models_library.users import UserID
+from pydantic import parse_raw_as
+from simcore_service_director_v2.core.settings import AppSettings
+from simcore_service_director_v2.models.domains.comp_pipelines import CompPipelineAtDB
+from simcore_service_director_v2.models.domains.comp_tasks import CompTaskAtDB
+from simcore_service_director_v2.models.schemas.comp_tasks import TaskLogFileGet
+
+pytest_simcore_core_services_selection = [
+    "postgres",
+]
+pytest_simcore_ops_services_selection = [
+    "adminer",
+]
+
+
+def get_app(async_client: httpx.AsyncClient) -> FastAPI:
+    # pylint: disable=protected-access
+    app = async_client._transport.app  # type: ignore
+    assert app
+    assert isinstance(app, FastAPI)
+    return app
+
+
+@pytest.fixture
+def mock_env(
+    mock_env: None,  # sets default env vars
+    postgres_host_config,  # sets postgres env vars
+    monkeypatch: pytest.MonkeyPatch,
+):
+    # overrides mock_env
+    monkeypatch.setenv("S3_ENDPOINT", "endpoint")
+    monkeypatch.setenv("S3_ACCESS_KEY", "access_key")
+    monkeypatch.setenv("S3_SECRET_KEY", "secret_key")
+    monkeypatch.setenv("S3_BUCKET_NAME", "bucket_name")
+    monkeypatch.setenv("S3_SECURE", "false")
+
+    monkeypatch.setenv("DIRECTOR_V2_POSTGRES_ENABLED", "True")
+
+
+@pytest.fixture
+def client(async_client: httpx.AsyncClient, mocker) -> httpx.AsyncClient:
+    # overrides client
+    app = get_app(async_client)
+
+    settings: AppSettings = app.state.settings
+    assert settings
+    print(settings.json(indent=1))
+
+    return async_client
+
+
+@pytest.fixture
+def mocked_nodeports_storage_client(mocker, faker: Faker) -> dict[str, mock.MagicMock]:
+    # NOTE: mocking storage API would require aioresponses since the access to storage
+    # is via node-ports which uses aiohttp-client! In order to avoid adding an extra
+    # dependency we will patch storage-client functions in simcore-sdk's nodeports
```

Comment:
yes that is also what is done for isntance in [services/director-v2/tests/unit/with_dbs/test_utils_dask.py]

Reply:
Yes, I took it from there but mocked the functions i needed.